Feat implement get internet speed dart

GunjanDhanuka opened the pull request:
:heavy_check_mark: Dart implementation of the getInternetSpeed function as mentioned in #89 . This function uses Speedtest CLI to check the download and upload speed of the server. The results are printed as the output of the function in the units of Mbps.

lohanidamodar commented on dart/getInternetSpeed/README.md:
```diff
@@ -0,0 +1,41 @@
+## 📝 Environment Variables
+No environment variables required for this one
+
+
```
Remove extra line
```suggestion
```

lohanidamodar commented on dart/getInternetSpeed/README.md:
````diff
@@ -0,0 +1,41 @@
+
+```bash
+$ cd demos-for-functions/dart/getInternetSpeed
+$ dart pub get
````
need an step to update PUB_CACHE environment to make sure pub dependencies are kept in `.appwrite` folder and folder structure should contain `.appwrite/` folder. Check existing dart function demo.

GunjanDhanuka replied:
Done! Thanks a lot! I am new to contributing too appwrite hence these details got missed :sweat_smile:

lohanidamodar commented on dart/getInternetSpeed/main.dart:
```diff
@@ -0,0 +1,57 @@
+import 'dart:convert';
+import 'dart:io';
+
+// void main() {
```
Please remove if this is not necessary. Thank you.

GunjanDhanuka replied:
Done!

lohanidamodar commented on dart/getInternetSpeed/pubspec.yaml:
```diff
@@ -0,0 +1,9 @@
+  sdk: ">=2.12.0 <3.0.0"
+
+dependencies:
+  dart_appwrite: ^0.6.1
```
Is this package used anywhere?

GunjanDhanuka replied:
No it isn't. I have updated it! Thanks!

GunjanDhanuka commented:
> heavy_check_mark Dart implementation of the getInternetSpeed function as mentioned in #89 . This function uses Speedtest CLI to check the download and upload speed of the server. The results are printed as the output of the function in the units of Mbps.

I have made the corrections as suggested! Please review it

lohanidamodar reviewed (changes requested):
@GunjanDhanuka please there is an unintended removal of `python/.gitkeep`, please fix it. Thank you

Meldiron commented:
Also, please rename the folder from `getInternetSpeed` to `get-internet-speed` to stay  consistent with the rest of the repo.

GunjanDhanuka commented:
> @GunjanDhanuka please there is an unintended removal of `python/.gitkeep`, please fix it. Thank you

That file was an empty file created since I mistakenly started working on Python implementation on the same branch. That is why I deleted that file. Do I still need to add that empty file?

GunjanDhanuka commented:
> Also, please rename the folder from `getInternetSpeed` to `get-internet-speed` to stay consistent with the rest of the repo.

Done! :+1:

GunjanDhanuka commented:
> > @GunjanDhanuka please there is an unintended removal of `python/.gitkeep`, please fix it. Thank you
> 
> That file was an empty file created since I mistakenly started working on Python implementation on the same branch. That is why I deleted that file. Do I still need to add that empty file?

Nevermind, I added the ```python/.gitkeep``` file again since it was in the ```master``` branch

lohanidamodar commented on dart/get-internet-speed/README.md:
```diff
@@ -0,0 +1,43 @@
+# 📧 getInternetSpeed
```
```suggestion
# 📧 Get Internet Speed
```

GunjanDhanuka replied:
Done!

lohanidamodar commented on dart/get-internet-speed/pubspec.yaml:
```diff
@@ -0,0 +1,9 @@
+environment:
+  sdk: ">=2.12.0 <3.0.0"
+
+dependencies:
```
```suggestion
```
As there are not dependencies we can remove this. Make sure it works after removing this.

GunjanDhanuka replied:
Done!

lohanidamodar reviewed (changes requested):
@GunjanDhanuka great work. Some changes.
Also there are two copies of every file in two different folders. Make sure to remove duplicates!

GunjanDhanuka commented:
> @GunjanDhanuka great work. Some changes. Also there are two copies of every file in two different folders. Make sure to remove duplicates!

@lohanidamodar I have made the requested changes! Can you please review them?

lohanidamodar commented:
@GunjanDhanuka please also add screenshot of successful execution and output.

Meldiron reviewed (changes requested):
Please include a screenshot of successful execution and its output from Appwrite Console. Without that, we cannot proceed to merge the PR.

christyjacob4 commented:
@GunjanDhanuka Closing this PR since there hasn't been any activity in a while. Feel free to reopen it when you're ready to start working on it again.
